manifest.json

```diff
@@ -10,16 +10,21 @@
     "url": "http://www.minetest.net/",
     "license": "MIT",
     "maintainer": {
-        "name": "Luc Didry",
-        "email": "luc@framasoft.org",
-        "url": "https://framasoft.org"
+        "name": "rafi59",
+        "email": "rafi59_dev@srvmaison.fr.nf",
+        "url": ""
     },
+    "previous_maintainers": {
+    	"name": "Luc Didry",
+    	"email": "luc@framasoft.org",
+	"url": "https://framasoft.org"
+},
     "requirements": {
-        "yunohost": ">> 2.6.4"
+        "yunohost": ">= 3.5"
     },
-    "multi_instance": true,
+    "multi_instance": false,
```

Comment:
Maybe, if you replace all `mintest` by `$app` in script, you can leave this to true... But, does that make sense to have multiple instances of this game?